importers/aircraft_csv_format_importer.py

```diff
@@ -0,0 +1,157 @@
+            name="Aircraft CSV Format Importer",
+            validation_level=constants.BASIC_LEVEL,
+            short_name=" Aircraft CSV Importer",
+            default_privacy="Public",
+            datafile_type="CSV",
+        )
+        self.text_label = None
+
+    def can_load_this_type(self, suffix):
+        return suffix.upper() == ".CSV"
+
+    def can_load_this_filename(self, filename):
+        return True
+
+    def can_load_this_header(self, header):
+        return header.startswith("Date(Uk)")
+
+    def can_load_this_file(self, file_contents):
+        return True
+
+    def _load_this_line(self, data_store, line_number, line, datafile, change_id):
+        # Skip the header
+        if line_number == 1:
+            return
+
+        tokens = line.tokens(line.CSV_TOKENISER, ",")
+        if len(tokens) <= 1:
+            # The last line may be empty, don't worry
+            return
+        if len(tokens) < 8:
+            # CSV example file has 9 columns
+            self.errors.append(
+                {self.error_type: f"Error on line {line_number}. Not enough tokens: {line}"}
+            )
+            return
+
+        # Separate token strings
+        date_token = tokens[0]
+        time_token = tokens[1]
+        # lat_token = tokens[2]  # Currently not used
+        # long_token = tokens[3]  # Currently not used
+        lat_degrees_token = tokens[4]
+        long_degrees_token = tokens[5]
+        altitude_token = tokens[6]
+        speed_token = tokens[7]
+        course_token = tokens[8]
+
+        # Check Date is in correct format
+        if len(date_token.text) != 10:
+            self.errors.append(
+                {
+                    self.error_type: f"Error on line {line_number}. Date format '{date_token.text}' "
+                    f"should be 10 figure data"
+                }
+            )
+            return
+
+        # Times always in Zulu/GMT
+        if len(time_token.text) != 8:
+            self.errors.append(
+                {
+                    self.error_type: f"Line {line_number}. Error in Time format '{time_token.text}'."
+                    "Should be HH:mm:ss"
+                }
+            )
+            return
+
+        timestamp = self.parse_timestamp(date_token.text, time_token.text)
+        if timestamp:
+            combine_tokens(date_token, time_token).record(self.name, "timestamp", timestamp)
+        else:
+            # Skip line if invalid timestamp
+            self.errors.append(
+                {self.error_type: f"Line {line_number}. Error in timestamp parsing."}
+            )
+            return
+
+        # And finally store it - Check how to do this as currently don't have a platform name
+        platform = self.get_cached_platform(data_store, platform_name=None, change_id=change_id)
+        sensor_type = data_store.add_to_sensor_types("Location-Satellite", change_id=change_id).name
+        privacy = get_lowest_privacy(data_store)
```

Comment:
We can't use the lowest privacy here, we need to get it from the analyst (via the resolver)

Reply:
Hi Ian, 
Currently all other importers either use this to set function above to set the privacy or they just don't appear to set it at all. The current exception that I can find is the `nmea_importer.py` which uses the following instead: `default_privacy="Private"`
Do you have any suggestions on how best to handle this?

Reply:
You should be able to just pass `privacy=None` to the `get_sensor` call, and the resolver will ask the user to enter a privacy. (Please test this by running pepys_import _without_ `--resolver default` to check it asks for a privacy, as I may be wrong).